app/models/price_table.rb

```diff
@@ -1,5 +1,5 @@
 class PriceTable < ApplicationRecord
-    validates_presence_of :trainees, :price_point
+    validates_presence_of :price_point, :trainee_1, :trainee_3, :trainee_4, :trainee_5, :trainee_6
```

Comment:
Shouldn't we have 7 trainees at the most?

Reply:
Discussed with Rupert and updated accordingly